SU2_CFD/src/drivers/CMultizoneDriver.cpp

```diff
@@ -139,13 +139,13 @@ CMultizoneDriver::~CMultizoneDriver(void) {
 void CMultizoneDriver::StartSolver() {
 
   /*--- Find out the minimum of all references times and then set each zone to this (same) value.
-   * (To ensure that all zones run synchronously in time, be it a dimensional or non-dimensionalized one.) ---*/
+        To ensure that all zones run synchronously in time, be it a dimensional or non-dimensionalized one. ---*/
 
   su2double Time_Ref = config_container[ZONE_0]->GetTime_Ref();
 
   for (iZone = 1; iZone < nZone; iZone++) {
     if (config_container[iZone]->GetTime_Ref() < Time_Ref)
-      Time_Ref = config_container[iZone]->GetTime_Ref();
+      Time_Ref = min(Time_Ref, config_container[iZone]->GetTime_Ref());
```

Comment:
closer look

Reply:
🕵️

Reply:
The "if" is redundant after your change.

Reply:
hmm, one could get the impression I have the stupid.

Reply:
got rid of the if. Thanks for the review and catching all these inadequacies